src/igvfd/report.py

```diff
@@ -56,20 +62,37 @@ def _convert_camel_to_snake(type_str):
 @view_config(route_name='report_download', request_method='GET')
 def report_download(context, request):
     downloadtime = datetime.datetime.now()
-
-    types = request.params.getall('type')
-    if len(types) != 1:
-        msg = 'Report view requires specifying a single type.'
-        raise HTTPBadRequest(explanation=msg)
-
+    request.GET['limit'] = '0'
+    fr = FieldedResponse(
+        _meta={
+            'params_parser': ParamsParser(request)
+        },
+        response_fields=[
+            MultipleTypesReportWithFacetsResponseField(
+                default_item_types=DEFAULT_ITEM_TYPES,
+                reserved_keys=RESERVED_KEYS,
+            ),
+            ColumnsResponseField(),
+
+        ]
+    )
```

Comment:
We should be calling the existing view instead of recreating it here. Something like `response = request.embed(f'/multitype-report/{original_query_string_with_limit_0_added}')`

Reply:
Got it. Thanks!